flixel/text/FlxText.hx

```diff
@@ -1066,4 +1048,32 @@ abstract FlxTextAlign(String) from String
 	var CENTER = "center";
 	var RIGHT = "right";
 	var JUSTIFY = "justify";
+	
+	#if !openfl_legacy
+	public static function fromOpenFL(tf:TextFormatAlign):FlxTextAlign
+	#else
+	public static function fromOpenFL(tf:String):FlxTextAlign
+	#end
+	{
+		//ifs instead of a switch b/c you can only switch on lower-case variable names and TextFormatAlign are all caps constants in legacy
```

Comment:
Hmm, you're right, now it's suddenly working again. 

The error *was* "Capture variables must be lower-case"